Community/README.md

```diff
@@ -0,0 +1,48 @@
+# Community
+
+Entrypoint for getting onboarded.
+
+## Tools used by the community
+
+- Instant messaging
+  - [Matrix](https://matrix.to/#/!TiDqlLmEUaXqTemaLc:matrix.org?via=matrix.org)
+  - [Nextcloud Talk](https://scs.sovereignit.de/nextcloud) (Not used)
+  - [Fosstodon](https://fosstodon.org/@sovereigncloudstack) (Only one-directional)
+- Annoucements
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+  - [RSS Feed](https://scs.community/feed.xml)
+  - [Footodon](https://fosstodon.org/@sovereigncloudstack)
+  - [Twitter](https://twitter.com/scs_osballiance)
+  - [LinedIn](https://www.linkedin.com/showcase/sovereigncloudstack)
+  - [Website](https://scs.community)
+- Video-Calls
+  - [Jitsi](https://conf.scs.koeln:8443/SCS-TECH)
+- Groupware/Fileshare/Organizational Stuff
+  - [Nextcloud](https://scs.sovereignit.de/nextcloud)
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+- Task distribution
+  - [Wekan](https://scs.sovereignit.de/wekan)
+  - [GitHub](https://github.com/SovereignCloudStack)
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+- Guides
+  - [GitHub](https://github.com/SovereignCloudStack)
+  - [Contributor Guide](https://scs.community/docs/contributor/)
+
+### Proposal to improve (Just my two cents)
+
+- reduce usage of mailing lists drastically
+- reduce amount of mailing lists
+- use single source of annoucement
+- don't spam the mailing lists with complete annoucements, but rather the link to the annoucement!
+- ditch Wekan and move to GitHub to reduce tool clutter
+- provide ONE single URL for all services with proper subdomains:
+  - nextcloud.sovereignit.de
+  - lists.sovereignit.de
+  - jitsi.sovereignit.de
+  - sovereignit.de (website)
+  - I've never understood why there is scs.community and scs.sovereignit.de
+- I would love to have a single tool for chat and video calls (slack, zulip, rocket chat, mattermost, element matrix, microsoft teams, i don't care)
+- overall documentation is too much distributed. I would expect this "https://scs.community/docs/contributor/" to be available as a menu point directly in "https://scs.community/". AND: I would expect it to be in the same design style. Would make things feel much more smooth.
```

Comment:
This is something we want to tackle within the next months. IMHO all documentation should be reside in the `SovereignCloudStack/Docs` repository. This repository should then be used to generate a website by using a static site generator. I'll have a look if we can easily match the `website` Jekyll framework with the `Docs` repository.